addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -71,14 +76,14 @@
                                 </DropdownItem>
                             </t>
                         </Dropdown>
-                    </td>
-                </tr>
+                    </div>
+                </div>
                 <!-- Add / remove selection labels -->
-                <tr t-if="state.propertyDefinition.type === 'selection'">
-                    <td class="o_td_label align-top pe-2">
+                <div t-if="state.propertyDefinition.type === 'selection'" class="d-flex align-items-center">
+                    <div class="w-25 align-self-start me-2">
                         <b>Values</b>
```

Comment:
Ideally I would like this card to inherit the formView design. Could you use the same structure and classes?